OCMapper/Source/Mapping Provider/In Code Mapping/InCodeMappingProvider.m

```diff
@@ -72,6 +80,18 @@ - (void)mapFromDictionaryKey:(NSString *)dictionaryKey toPropertyKey:(NSString *
 	[self.mappingDictionary setObject:info forKey:key];
 }
 
+- (void)mapFromPropertyKey:(NSString *)propertyKey toDictionaryKey:(NSString *)dictionaryKey forClass:(Class)class {
+    ObjectMappingInfo *info = [[ObjectMappingInfo alloc] initWithDictionaryKey:dictionaryKey propertyKey:propertyKey andObjectType:nil];
+    NSString *key = [self uniqueKeyForClass:class andKey:propertyKey];
+    [self.inverseMappingDictionary setObject:info forKey:key];
+}
+
+- (void)mapFromPropertyKey:(NSString *)propertyKey toDictionaryKey:(NSString *)dictionaryKey forClass:(Class)class withTransformer:(MappingTransformer)transformer {
+    ObjectMappingInfo *info = [[ObjectMappingInfo alloc] initWithDictionaryKey:dictionaryKey propertyKey:propertyKey andTransformer:transformer];
+    NSString *key = [self uniqueKeyForClass:class andKey:propertyKey];
+    [self.inverseMappingDictionary setObject:info forKey:key];
+}
+
 - (void)setDateFormatter:(NSDateFormatter *)dateFormatter forProperty:(NSString *)property andClass:(Class)class
```

Comment:
Add inverse date formatter, and take automaticallyGenerateInverseMapping into consideration